corehq/apps/sms/migrations/0049_auto_enable_turnio_ff.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.24 on 2021-06-10 09:13
+
+from django.db import migrations
+from corehq.messaging.smsbackends.turn.models import SQLTurnWhatsAppBackend
+from corehq.toggles import TURN_IO_BACKEND
+
+
+def auto_enable_turnio_ff_for_certain_domains(apps, schema_editor):
+    for backend in SQLTurnWhatsAppBackend.active_objects.all():
+        domain = backend.domain
+        TURN_IO_BACKEND.set(item=domain, enabled=True, namespace='domain')
```

Comment:
You should use `toggles.NAMESPACE_DOMAIN` instead of hardcoding the string value.